homeassistant/components/template/strings.json

```
@@ -176,13 +209,25 @@
           "set_temperature": "Actions on set color temperature"
         },
         "data_description": {
-          "device_id": "[%key:component::template::common::device_id_description%]"
+          "device_id": "[%key:component::template::common::device_id_description%]",
+          "state": "The light is `on` if the template evaluates as `True`, `yes`, `on`, `enable` or a positive number. Any other value will render it as `off`.",
+          "turn_off": "Defines actions to run when the light is turned off.",
+          "turn_on": "Defines actions to run when the light is turned on.",
+          "level": "Defines a template to get the brightness of the light.  Valid values are 0 to 255.",
```

Nitpicking … double space here:

```suggestion
          "level": "Defines a template to get the brightness of the light. Valid values are 0 to 255.",
```